packages/ui/src/forum/components/category/storybook-helpers.ts

```diff
@@ -0,0 +1,37 @@
+import { sub } from 'date-fns'
+import faker from 'faker'
+
+import { asBlock } from '@/common/types'
+import { ForumModerator, ForumPost, ForumThread } from '@/forum/types'
+import { getMember } from '@/mocks/helpers'
+
+export const asModerator = (hasHandle = true, hasAvatar = false) => {
+  return (index: number): ForumModerator => ({
+    id: String(index),
+    membershipId: '0',
+    handle: hasHandle ? faker.name.firstName() : undefined,
+    avatar: hasAvatar ? faker.image.avatar() : undefined,
+  })
+}
+
+export const asPost = (text: string): ForumPost | undefined => {
```

Comment:
Maybe those helpers should be named a bit differently to not confuse them with standard `as...()` casting functions.

I'd go with something like `asStorybookPost()` & `asStorybookModerator()` or only `asStoryPost()`, WDYT?

Reply:
Thanks, sorry I had not seen the review while dealing with the conflict. Your right I will rename these